source/documentation/using_the_api/datatypes.md

```diff
@@ -0,0 +1,26 @@
+## Datatypes
+
+The simplest way to find a field’s datatype is to use the record in the appropriate field register. This is of the form `https://field.{domain}/records/{field}`, where `{domain}` is the [domain of your register from its base URL](#find-the-base-url-for-register-s
+), and `{field}` is the field you’re interested in. 
+
+For example, the [datatype of the `local-authority-type` field in the `local-authority-eng` register is `string`](https://field.register.gov.uk/records/local-authority-type.json). 
+
+Go to `https://datatype.{domain}/records/{datatype}` to find the definition of any given datatype of interest (`{datatype}`). 
+
+### How to resolve foreign keys
```

Comment:
I don't think this section belongs to "Datatypes"

Reply:
Yes, I thought it may not (same for the CURIEs section); that's why I added a pre-emptive comment under the 'Datatypes' h2.